apps/api/src/app/events/dtos/trigger-event-request.dto.ts

```diff
@@ -1,7 +1,6 @@
 import { ArrayMaxSize, ArrayNotEmpty, IsArray, IsDefined, IsObject, IsOptional, IsString } from 'class-validator';
 import { ApiExtraModels, ApiProperty, ApiPropertyOptional, getSchemaPath } from '@nestjs/swagger';
-import { TriggerRecipientSubscriber, TriggerRecipients } from '@novu/node';
```

Comment:
We had a lot of dependencies of types from the Node package so I moved them to the Shared package as it would be expected.